tvdbsimple/user.py

```diff
@@ -165,6 +162,10 @@ def __init__(self, user, key, **kwargs):
         self._FILTERS = {}
         self.update_filters(**kwargs)
 
+        self._PAGES = -1
+        self._PAGES_LIST = {}
+        self._FILTERS = {}
```

Comment:
Same here..but in this case _FILTERS is already set so you could simply drop this line..